src/app/modules/shared.module.ts

```diff
@@ -0,0 +1,23 @@
+import { NgModule } from '@angular/core';
+import { LoginService } from '../services/login.service';
+import { LoginComponent } from '../pages/login';
+import { FormsModule } from '@angular/forms';
+import {
+  RouterModule,
+  PreloadAllModules
+} from '@angular/router';
+
+@NgModule({
+  declarations: [
+    LoginComponent,
```

Comment:
Osobiście uważam, że nie ma sensu tworzyć oddzielnych modułów dla header, footer. To nie są jakieś bardzo skomplikowane komponenty z ogromną logiką, za to są współdzielone na wszystkich stronach i dlatego widzę je w module shared ;) Dodanie nowych modułów, które mają tylko jeden komponent powoduje tylko narzut plików.

No, ale rób jak chcesz. 

PS. Moduł do autoryzacji z serwisem logowania i komponentami z formularzem logowania już jest bardziej zasadny.